cumulusci.yml

```diff
@@ -836,9 +836,13 @@ flows:
                 # Enables RD2 in Custom Settings. Do this before enabling CRLP so it does not create a rollup for NextDonationDate.
                 options:
                     apex: >
-                        npe03__Recurring_Donations_Settings__c rdSettings = npe03__Recurring_Donations_Settings__c.getOrgDefaults();
-                        rdSettings.%%%NAMESPACE%%%isRecurringDonations2Enabled__c = true;
-                        upsert rdSettings;
+                        Map<String, Object> params = new Map<String, Object>{ 'ScheduleJobs' => true };
+                        String shouldScheduleJobs = '%%%PARAM_1%%%';
+                        if (!String.isEmpty(shouldScheduleJobs)) {
+                            params.put('ScheduleJobs', Boolean.valueOf(shouldScheduleJobs));
+                        }
+                        Callable apiClass = new %%%NAMESPACE%%%Callable_Api();
+                        apiClass.call('settings.enableenhancedrecurringdonations', params);
```

Comment:
This doesn't have to be lowercase. It's probably easier to read if `Settings.EnableEnhancedRecurringDonations` is in camel-case.